Mathlib/RingTheory/LocalProperties/Basic.lean

```diff
@@ -240,8 +240,8 @@ theorem RingHom.PropertyIsLocal.ofLocalizationSpan (hP : RingHom.PropertyIsLocal
   rintro ⟨_, r, hr, rfl⟩
   rw [← IsLocalization.map_comp (M := Submonoid.powers r) (S := Localization.Away r)
     (T := Submonoid.powers (f r))]
-  apply hP.StableUnderCompositionWithLocalizationAway.right _ r
-  exact hs' ⟨r, hr⟩
+  · apply hP.StableUnderCompositionWithLocalizationAway.right _ r
+    exact hs' ⟨r, hr⟩
```

Comment:
Are there actually two goals here, and closing the first one discharges the second as a side effect?

Reply:
Yes, the other goal is
```lean
Submonoid.powers r ≤ Submonoid.comap f (Submonoid.powers (f r))
```
and it disappears once the previous goal is closed.